ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -735,7 +746,8 @@ runThreadNetwork ThreadNetworkArgs
             { tracers             =
                 -- traces the node's local events other than those from the
                 -- ChainDB
-                instrumentationTracers <> nullDebugTracers
+                instrumentationTracers <>
+                showTracers (decorateId coreNodeId debugTracer)
```

Comment:
TODO The test suite not generate any traces by default, so this line shouldn't be merged as-is. (Let me know if you disagree.)